src/spectrecoff/Table.fs

```diff
@@ -103,6 +103,6 @@ let customTable (layout: TableLayout) (headers: Header list) (rows: Row list) =
 let table =
     customTable defaultTableLayout
 
-let toConsole (table: Table) =
+// Output methods
```

Comment:
I thought I removed this comment, did you re-add it on purpose?

Reply:
No, sry, adn will be removed in follou-up PR that I am preparing already. Since, you know, the whole 'toConsole' will dissapear :P